solutions/usaco-815.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: usaco-815
+source: USACO Gold 2018 Febuary
+title: Taming the Herd
+author: Shijie Ren
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_gold_feb18.html)
+
+## Explanation
+
+Let $\texttt{dp}[i][j]$ be the minimum answer of first $j$ days with $i$
+breakouts.
+
+On the $j$-th day, there's 3 cases:
```

Comment:
why does each case result in the said dp relation?

Reply:
sorry I can't understand you very well, so what should I add

Reply:
like $dp[...] = dp[...]$ because [whatever reason]